docs/pages/machine-id/guides/applications.mdx

```diff
@@ -0,0 +1,281 @@
+[Getting Started Guide](../getting-started.mdx) to familiarize yourself with
+Machine ID. You'll also need `tctl` access to initially configure the bot.
+
+(!docs/pages/includes/tctl.mdx!)
+
+Lastly, ensure the `tbot` binary is installed on your Machine ID client system.
+Refer to our [Installation guide](../../installation.mdx) for more information.
+
+## Step 1/3. Create a Machine ID bot and assign permissions
+
+In this example you'll start by creating a bot named "example" and assign it a
+role granting access to some set of apps by label.
+
+First, on a node with `tctl` access (such as your local machine), create a file named `role.yaml` with the
+following content:
+
+```yaml
+kind: role
+metadata:
+  name: machine-id-app-role
+version: v5
+spec:
+  allow:
+    app_labels:
+      '*': '*'
+```
+
+This role allows access to all apps. The label selector here can be used to
+restrict access as desired and supports both wildcards and regular expressions.
+Refer to the [Access Controls Reference](../../access-controls/reference.mdx#roles)
+for more information.
+
+Once finished, create the role on your Teleport cluster:
+```code
+$ tctl create -f role.yaml
+```
+
+## Step 2/3. Configure and start Machine ID
+
+First, on a node with `tctl` access, create a new Machine ID bot using the role
+created in the previous step:
+```code
+$ tctl bots add example --roles=machine-id-app-role
+```
+
+The creates a bot named `example` with permissions to access the apps as you
+configured above. Be sure to note the bot join token and CA PIN.
+
+Next, on the bot node, create a Machine ID configuration file at
+`/etc/tbot.yaml`:
+
+<ScopedBlock scope={["cloud"]}>
+```yaml
+auth_server: "example.teleport.sh:443"
+onboarding:
+  join_method: "token"
+  token: "00000000000000000000000000000000"
+  ca_pins:
+  - "sha256:1111111111111111111111111111111111111111111111111111111111111111"
+storage:
+  directory: /var/lib/teleport/bot
+destinations:
+  - directory: /opt/machine-id
+    app: grafana-example
+```
+</ScopedBlock>
+<ScopedBlock scope={["oss","enterprise"]}>
+```yaml
+auth_server: "teleport.example.com:443"
+onboarding:
+  join_method: "token"
+  token: "(=presets.tokens.first=)"
+  ca_pins:
+  - "(=presets.ca_pin=)"
+storage:
+  directory: /var/lib/teleport/bot
+destinations:
+  - directory: /opt/machine-id
+    app: grafana-example
+```
+</ScopedBlock>
+
+Be sure to configure the `token` and `ca_pins` fields to match the output from
+`tctl bots add ...`, and set `app` to match the desired name as shown in
+`tsh apps ls`. For this example, we're using the `grafana-example` app.
+
+Machine ID also allows you to use Linux ACLs to control access to certificates
+on disk. You will use this to ensure only your application has access to the
+short-lived certificates Machine ID uses.
+
+We'll work with the assumption you will be running Machine ID as the Linux user
+`teleport` and your application as the Linux user `app`. Create and initialize
```

Comment:
> We'll work with the assumption you will be running Machine ID as the Linux user
> `teleport` and your application as the Linux user `app`.

I think we should make this more explicit that "your application" refers to the app you're using to access an upstream application protected by Teleport, not the upstream application itself (assuming that's correct).

Later, the guide says this application is an "automated service." I think that term works, but we should also clarify earlier in the guide that Machine ID will be providing credentials to an automated service that is connecting to your application.